src/tests/test_buildercore_trop.py

```diff
@@ -1217,6 +1217,11 @@ def test_docdb(self):
                                  'Value': 'project-with-docdb--prod'},
                                 {'Key': 'Project',
                                  'Value': 'project-with-docdb'}]},
+                      'VPCSecurityGroups': {
```

Comment:
Not sure what the exact value for this object should be, or if it should be omitted and fed in using the `with patch` statement below. Any help or pointers appreciated.

Reply:
that specific test can be run with:

```bash
cd /path/to/builder
source venv/bin/activate
./.run-tests.sh ./src/tests/test_buildercore_trop.py::TestBuildercoreTrop::test_docdb
```

Scripts with a preceding '.' typically assume the activation of a python virtualenv.

`VpcId` is pulled from the test project file. The test project file has a lot of values similar to the project file in normal use but some are dummied up or missing etc. You can find them in the test fixtures.